ml3d/metrics/mAP.py

```diff
@@ -246,6 +246,9 @@ def mAP(pred,
                 prec[ti] = tp_acc / (tp_acc + fp_acc)
                 prec[ti] = np.max(prec[ti:], axis=-1)
 
-            mAP[i, j] = np.sum(prec[::4]) / 11 * 100
+            if len(prec[::4]) < 11:
```

Comment:
I don't think we should have an "11" here. That constant should be defined somewhere.